config/app/envs/prod.yaml

```diff
@@ -301,7 +287,9 @@ servers:
       name: view
 
 ## RANGE BEGINS
-{{$Name := .ApplicationName}}{{ range $index, $Num := .ServerCount }}
+{{$Name := .ApplicationName}}
+{{$KeyStorePW := .KeyStorePassword}}
+{{ range $index, $Num := .ServerCount }}
```

Comment:
Do you really need to introduce a new/local variable for each loop, even though you are using the same global var in the loop?

Reply:
unfortunately, based on what i've seen.  I guess i could try to duplicate the same vars within the ServerCount slice instead... have each var duplicated within the EnvTemplate struct, hande it there.

Reply:
Ok, that's fine, was just wondering why you were doing that.